app/src/main/java/ca/bc/gov/bchealth/ui/healthrecord/HealthRecordPlaceholderViewModel.kt

```diff
@@ -57,3 +61,8 @@ data class PatientHealthRecord(
     val totalRecord: Int,
     val authStatus: AuthenticationStatus
 )
+
+enum class Availability {
```

Comment:
Rename enum class name to more relevant name.

Reply:
Taken care